safenode/src/domain/wallet/local_store.rs

```diff
@@ -496,6 +519,72 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn created_dbc_to_file_can_be_deposited() -> Result<()> {
+        // Bring in the necessary traits.
+        use super::{DepositWallet, SendWallet};
+
+        let sender_root_dir = create_temp_dir()?;
+        let sender_root_dir = sender_root_dir.path().to_path_buf();
+
+        let mut sender = LocalWallet::load_from(&sender_root_dir).await?;
+        let sender_dbc = create_genesis_dbc(&sender.key).expect("Genesis creation to succeed.");
+        sender.deposit(vec![sender_dbc]);
+
+        let send_amount = 100;
+
+        // Send to a new address.
+        let recipient_root_dir = create_temp_dir()?;
```

Comment:
You could consider using the `assert_fs` crate for working with and creating with temporary directories. It's really cool and useful. It drops any directories when they go out of scope, whereas this code would leave them around after the test runs.

Reply:
Done!